src/main/java/com/marginallyclever/makelangelo/plotter/plottercontrols/MarlinInterface.java

```diff
@@ -271,20 +283,49 @@ public void sendESTOP() {
 		chatInterface.sendCommand("M112");
 	}
 
+
+	private void processActionCommand(String actionCommand) {
+		if(actionCommand.startsWith(PROMPT_BEGIN)) {
+			promptDialog.setPromptMessage(actionCommand.substring(PROMPT_BEGIN.length()));
+			promptDialog.clearPrompts();
+		} else if(actionCommand.startsWith(PROMPT_CHOICE))
```

Comment:
can you add `{` and `}` to if statement even when there is only one line, it will improve readability

Reply:
ok